powerline_prompt.lua

```diff
@@ -6,15 +6,70 @@
 local promptValueFull = "full"
  -- "folder" for folder name only like System32
 local promptValueFolder = "folder"
+
+local promptValueNone = "none"
+
  -- default is promptValueFull
 local promptValue = promptValueFull
 
+local pathToProject = nil
+
+
 local function get_folder_name(path)
 	local reversePath = string.reverse(path)
 	local slashIndex = string.find(reversePath, "\\")
 	return string.sub(path, string.len(path) - slashIndex + 2)
 end
 
+local npmIcon = ""
```

Comment:
looks like this didn't get saved correctly?

Reply:
maybe encoding was messed somewhere along the way

Reply:
not really, set icomoon in your ide and in your cmder
![image](https://user-images.githubusercontent.com/4508208/39047958-ba5f0574-4493-11e8-991a-7f6a1d465e05.png)
Dont think git is correctly handling those symbols in the code snippets